### PR TITLE
[13.0][ADD] l10n_th_fonts

### PR DESCRIPTION
**Pull request by Saran440**
Add collection of fonts thai
- THSarabunNew
- AngsanaNew

### CODE REVIEW

File: l10n_th_fonts/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+{
+    "name": "Thai Localization - Fonts Thai",
+    "version": "13.0.1.0.0",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "summary": "Collection of all fonts thai",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/l10n-thailand",
+    "category": "Report",
+    "depends": [],
```
Comment (newtratip) [inline l10n_th_fonts/__manifest__.py]:
```suggestion
    "depends": ["base"],
```

**Review by newtratip — COMMENTED** [review]
Reviewed.

**Comment by Saran440** [comment]
@newtratip Fixed. Thank you for your reviewed.

**Review by newtratip — APPROVED** [review]

**Review by ps-tubtim — APPROVED** [review]

File: l10n_th_fonts/README.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure font thailand:
```
Comment (kittiu) [inline l10n_th_fonts/README.rst:41]:
```suggestion
To configure Thai font:
```

**Pull request merged**